cli/src/pcluster/templates/queue_group_stack.py

```diff
@@ -271,6 +273,9 @@ def _add_compute_resource_launch_template(
                                 "IntelHPCPlatform": "true" if self._config.is_intel_hpc_platform_enabled else "false",
                                 "CWLoggingEnabled": "true" if self._config.is_cw_logging_enabled else "false",
                                 "LogRotationEnabled": "true" if self._config.is_log_rotation_enabled else "false",
+                                "DetailedMonitoringEnabled": "true"
```

Comment:
Why do we need to pass it to compute nodes userdata? It is only needed as an attribute in  laugh template, not need to pass to cookbook